### PR TITLE
Build v2

### PR DESCRIPTION
**Pull request by ceolin**
- v2
 - Fix runner dependency from inspector
 - Fix the relations between node descriptions and resolver conffile
 - Use resolver builtin if building without resolver conffile
 - fix fbp parser dependency

### CODE REVIEW

File: src/lib/flow/sol-flow-buildopts.h.in [hunk 1]
```diff
@@ -38,4 +38,6 @@
 @SOL_BUILD_FLOW_MODULE_RESOLVER_CONFFILE@
 #ifdef SOL_BUILD_FLOW_MODULE_RESOLVER_CONFFILE
 extern struct sol_flow_resolver *sol_flow_resolver_conffile;
+#else
```
Comment (barbieri) [inline src/lib/flow/sol-flow-buildopts.h.in]: I think we should always define the builtin, not only if conffile is not defined. But @cmarcelo may have another opinion about it.

File: src/lib/flow/sol-flow-resolver.c [hunk 2]
```diff
@@ -108,12 +108,14 @@ builtins_resolve(void *data, const char *id, struct sol_flow_node_type const **t
...
 }
 
-static const struct sol_flow_resolver builtins_resolver = {
+static struct sol_flow_resolver builtins_resolver = {
     .api_version = SOL_FLOW_RESOLVER_API_VERSION,
     .name = "builtins_resolver",
     .resolve = builtins_resolve,
 };
 
+SOL_API struct sol_flow_resolver *sol_flow_resolver_builtins = &builtins_resolver;
```
Comment (barbieri) [inline src/lib/flow/sol-flow-resolver.c]: should keep builtins_resolver as const and declare here as const.

Comment (ceolin) [inline src/lib/flow/sol-flow-resolver.c]: this is a extern (firstly declared in buildopts.h), to set here as const i've to change the resolver as well.

**Comment by ceolin** [comment]
closed, fixes on https://github.com/solettaproject/soletta/pull/221

**Pull request closed without merging**